.github/workflows/3x.yml

```diff
@@ -46,7 +37,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ 'macos-latest' ]
-        php-versions: [ '8.0', '8.1' ]
+        php-versions: [ '7.4', '8.0' ]
```

Comment:
`php-versions: [ '7.4', '8.0', '8.1' ]`?

Reply:
Never mind.  I should have read further. :)

Reply:
No, this PR reverts all of the changes that added PHP 8.1 support, as those also introduced problems with 7.4.  The only thing this PR adds, beyond the removal of that commit, is testing on 7.4.